.env

```diff
@@ -1,8 +1,17 @@
+
+# Application
 APP_ENV=dev
 APP_VERSION=0.0.1-SNAPSHOT
-HUB=zufarexplainedit
-TAG=1.0.0
+APP_SERVER_PORT=8083
+
+# Docker
+DOCKER_HUB=shvedvp
+DOCKER_TAG=1.0.0
+DOCKER_COMPOSE_NETWORK=online-store-network
+
+# Datasource
 DATASOURCE_URL=jdbc:postgresql://postgresdb:5432/testdb?serverTimezone=UTC
```

Comment:
I would like to introduce 2 new variables:
DATASOURCE_HOST=posgresdb (or whatever)
DATASOURCE_DB_NAME=testdb
and delete DATASOURCE_URL
so afterwards we can use them in application properties in such way:
datasource:
     url: jdbc:postresql://${DATASOURCE_HOST}:${DATASOURCE_PORT}/${DATASOURCE_PORT}?serverTimezone=UTC
What do you think? From my point of view it will be more useful.

Reply:
Fixed